src/textformatter.cpp

```diff
@@ -160,7 +161,10 @@ std::vector<std::string> format_text_plain_helper(
 			static_cast<unsigned int>(type));
 
 		if (rxman && type != LineType::hr) {
-			rxman->quote_and_highlight(text, location);
+			// TODO: Propagate usage of StflRichText
+			auto x = StflRichText::from_quoted(text);
```

Comment:
Adding a comment here to make sure I don't miss this TODO

Reply:
It looks like this class handles both STFL richtext quoted text and regular non-STFL quoted text.
In case of unquoted text, `raw` is false and `rxman` is a nullptr (and for quoted, `raw` is true and `rxman` is a non-null pointer).

I prefer to refactor this class and resolve the TODO at a later moment (in a separate PR).